README.md

```
@@ -19,28 +23,46 @@
+# The README Project.
+
+<div align="center">
+
+  <em>The Startup & PhD Journey </em><br />
+  <sub><a href="about.md">The journey from the classroom to boardroom </a></sub>
+
+  [Ram Murmu ](https://github.com/rammurmu)
+
+👉 **[Watch Project](#-watch-tools-in-action)**
+
+### Open Source AI & ML Community 
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)